src/styles/blocks/card.scss

```diff
@@ -0,0 +1,38 @@
+.card {
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px;
+  box-shadow: 1px 1px 4px gray;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    padding-top: 33px;
+    padding-left: 20px;
+  }
+
+  &__title {
+    display: flex;
+    font-family: 'Roboto', sans-serif;
+    font-weight: 500;
+    font-style: normal;
+    font-size: 12px;
+    line-height: 14px;
+    width: 166px;
+    padding-top: 40px;
+    padding-left: 17px;
+  }
+
+  &__product_code {
+    display: flex;
+    font-family: 'Roboto', sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+    padding-top: 4px;
+    padding-left: 17px;
+  }
```

Comment:
The `&__product_code` block also has a fixed width and padding. Similar to the `&__title` block, consider using more responsive units for these properties.